### PR TITLE
Update SStream.c

### PR DESCRIPTION
**Pull request by Google-Autofuzz**
Merging in fixes from #1105 in Master to Next.

### CODE REVIEW

**Comment by aquynh** [comment]
awesome, but would you please fix the indentation? (we are using tabs for indentation)

**Comment by aquynh** [comment]
@catenacyber, do you have any comments? i remember you removed the check on LONG_MIN before because that was unnecessary, but this pull req wants to put it back again.

**Comment by catenacyber** [comment]
@aquynh yes the check is done afterwards.
So this PR does not look necessary to me.

**Comment by aquynh** [comment]
@Google-Autofuzz, we think this is not necessary, can you please confirm?

File: SStream.c [hunk 1]
```diff
@@ -55,7 +55,10 @@ void SStream_concat(SStream *ss, const char *fmt, ...)
 // print number with prefix #
 void printInt64Bang(SStream *O, int64_t val)
 {
-	if (val >= 0) {
+	if (val == LONG_MIN) {
+		return;
+	    }
```
Comment (radare) [inline SStream.c]: dont mix tabs and spaces for indenting

**Comment by Google-Autofuzz** [comment]
How so?

**Comment by catenacyber** [comment]
At least, you should remove the latter checks about `val == LONG_MIN` when you make the function return earlier

see https://github.com/aquynh/capstone/pull/1160

**Pull request closed without merging**